index.js

```diff
@@ -44,7 +44,12 @@ const { asyncForEach } = require('./tools/helpers'),
         let resourceFile = file.replace('.htl', '.js');
         let resourceFileFullPath = srcSpecFolder + resourceFile;
         // The reason to delete is to allow hot reloading to also rebuild components
-        delete require.cache[require.resolve('./' + resourceFileFullPath)];
+
+        // This line causes webpack to run into memory heap issues
+        // delete require.cache[require.resolve('./' + resourceFileFullPath)]; 

```

Comment:
Unfortunately, this is the line that is causing webpack to crash - I think webpack is getting confused where to look and starts loading unnecessary modules